src/common/controls/mixins/Copyright.js

```diff
@@ -0,0 +1,46 @@
+const CopyrightMixin = (Base) =>
+  class extends Base {
+    addCopyrightContainer(target) {
+      this.target = target;
+      this.copyrightElement = document.createElement('div');
+      this.copyrightElement.id = 'mb-copyrght';
+
+      Object.assign(this.copyrightElement.style, {
+        position: 'absolute',
+        bottom: 0,
+        right: 0,
+        fontSize: '10px',
+        padding: '0 10px',
+        backgroundColor: 'rgba(255, 255, 255, 0.5)',
+      });
+
+      this.target.appendChild(this.copyrightElement);
```

Comment:
if we choose to extend ol.Control . You 'll just have to call setMap(map) with the  generic content in src/common/Control.